README.md

```diff
@@ -8,75 +8,102 @@ We more than welcome contributions to KubeVirt documentation. Please reach out i
 
 ### Get started
 
-- Create fork of GitHub user-guide repo
+1. Create fork of GitHub user-guide repo
```

Comment:
I'm not fond of the static numbering vs bullets.  By the time I got to `7.` I no longer knew what step I was on.

What would you say to doing H4's?

Reply:
Sure that's pretty reasonable, numbered guides are probably best if you can stay <5! H4's work great for me.

- [ ] Change to H4 instead of numbered instructions